packages/kbn-check-mappings-update-cli/current_mappings.json

```diff
@@ -1645,6 +1645,9 @@
       "overrides": {
         "type": "flattened",
         "index": false
+      },
+      "keep_monitoring_alive": {
+        "type": "boolean"
```

Comment:
Looks like this file just got reformatted and that the only thing that will be added to mappings is this `keep_monitoring_alive` field (that I assume is needed for search)

Reply:
Might be time break out a sort algo on this maps keys here https://github.com/elastic/kibana/blob/167605e3469509a587607d4fbc7c96c3b4a4a580/packages/kbn-check-mappings-update-cli/src/current_mappings.ts#L32

I'll open an issue